test/test_nn.py

```diff
@@ -763,6 +763,23 @@ def test_Dropout3d(self):
         input = torch.Tensor(num_features, b, d, w, h)
         self._test_dropout(nn.Dropout3d, input)
 
+    def test_AlphaDropout(self):
+        # generate random tensor with zero mean and unit std
+        input = torch.randn(5000)
+
+        mean = input.mean()
+        std = input.std()
```

Comment:
Yes, but I left it like that in the event we might want to add support for a more general Alpha Dropout, which accepts a `mean` and `std` parameter. Also, because the samples are not infinitely large, I'd eventually need to increase the tolerance (or the tensor size) for the checks to consistently pass if I set those values to 0 and 1.